editor/config.json.bck

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
can you remove this file from the PR please @kalwalt

Reply:
yes absolutely!

Reply:
removed!